From 35852eafd298e27b9388b80ec018ffb1249f8538 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 16 Sep 2014 19:02:18 +0100
Subject: [PATCH 14/27] Add space before parens in function defns for
 readability.

---
 virt-what.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/virt-what.in b/virt-what.in
index f2c0b90..0770070 100644
--- a/virt-what.in
+++ b/virt-what.in
@@ -33,16 +33,16 @@ skip_qemu_kvm=false
 
 VERSION="@VERSION@"
 
-have_cpuinfo() {
+have_cpuinfo () {
     test -e "${root}/proc/cpuinfo"
 }
 
-fail() {
+fail () {
     echo "virt-what: $1" >&2
     exit 1
 }
 
-usage() {
+usage () {
     echo "virt-what [options]"
     echo "Options:"
     echo "  --help          Display this help"
-- 
2.10.2

