From ac098dd1513370145f10d6dd7c147b45168ef6a0 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 21 Apr 2015 12:12:45 +0100
Subject: [PATCH 17/27] xen-arm: Fix path in EXTRA_DIST.

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 514d229..8a47e96 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -201,7 +201,7 @@ EXTRA_DIST = \
 	tests/rhel5-xen-domU-pv/sys/hypervisor/compilation/compile_date \
 	tests/rhel5-xen-domU-pv/sys/hypervisor/uuid \
 	tests/xen-arm/proc/cpuinfo \
-	tests/xen-arm/proc/device-tree/compatible \
+	tests/xen-arm/proc/device-tree/hypervisor/compatible \
 	tests/xen-arm/proc/self/status \
 	tests/xen-arm/sbin/dmidecode \
 	tests/xen-arm/sbin/uname \
-- 
2.10.2

